net/wallet/Portfile

```diff
@@ -0,0 +1,239 @@
+                    and audit rules for documenting actions taken on that \
+                    data.  Objects of various types may be stored in the \
+                    wallet or generated on request and retrieved by \
+                    authorized users.  The wallet tracks ACLs, metadata, \
+                    and trace information.  It uses Kerberos \
+                    authentication.  One of the object types it supports \
+                    is Kerberos keytabs, making it suitable as a \
+                    user-accessible front-end to Kerberos kadmind with \
+                    richer ACL and metadata operations.
+homepage            http://eyrie.org/~eagle/software/wallet/
+
+platforms           darwin
+master_sites        http://archives.eyrie.org/software/kerberos/ \
+                    http://archives.eyrie.org/software/ARCHIVE/wallet/
+checksums           rmd160 188b3561fcffe99342fcfb1312b58df3f3d919b5 \
+                    sha256 676d3d6e407509fc9da1dd87d98fadc71920dabfbc4bdeb8cde5e2bc937268b8
+
+# Start with the dependencies we know we need
+depends_lib-append  port:kerberos5 \
+                    port:remctl
+
+# Add a dependency on Module::Build
+if {${perl5.major} != ""} {
+    depends_lib-append	port:p${perl5.major}-module-build
+}
+
+# wallet 1.3 does not support Perl paths other than /usr/bin/perl, nor does it
+# support perl binaries not named "perl".  The perl/Build.PL script is also
+# missing some testing prerequisites.
+# Since we are patching autoconf and automake files, we need autoreconf.
+patchfiles          patch-autogen.diff \
+                    patch-configure.ac.diff \
+                    patch-Makefile.am.diff \
+                    patch-README.diff \
+                    patch-perl-Build.PL.diff \
+                    patch-portable-system.h.diff \
+                    patch-rename-server-keytab-backend.diff \
+                    patch-rename-server-wallet-admin.diff \
+                    patch-rename-server-wallet-backend.diff \
+                    patch-rename-server-wallet-report.diff \
+                    patch-tests-client-full-t.in.diff \
+                    patch-tests-client-prompt-t.in.diff
+use_autoreconf      yes
+
+# Use configure, and pass in the paths to MacPorts kerberos5, remctl, and perl.
+configure.args      --enable-reduced-depends \
+                    --with-remctl=${prefix} \
+                    PATH_KRB5_CONFIG=${prefix}/bin/krb5-config \
+                    PERL=${prefix}/bin/perl${perl5.major} 
+
+# The Wallet client is just a couple of binaries, plus man pages and docs.
+variant client description 'Wallet client' {
+    long_description-append The wallet client uses the remctl protocol to \
+                            communicate with a wallet server.
+}
+
+# The Wallet server is entirely Perl, spawned by the remctl daemon.
+# TODO: Patch wallet remctl config files to use ${destroot}${prefix}
+variant server description 'Wallet server' {
```

Comment:
Rather than having variants for client and server, we should probably use subports here. Subports define a port with a separate name that shares the same Portfile except for the parts in `subport name { ... }` blocks or `if {$subport eq "name"} { ... }` conditions.

In this case I would use:
```
subport wallet-client {
  long_description-append ...
  notes-append ...
  destroot {
    # client-specific destroot
  }
}
```

This will make the various subports installable side-by-side and will cause our buildbot to build all of them. Note that you should probably install documentation in `${prefix}/share/doc/${subport}` then, to avoid file conflicts.